subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -131,6 +131,16 @@ uint8_t ll_scan_enable(uint8_t enable)
 		return BT_HCI_ERR_CMD_DISALLOWED;
 	}
 
+	scan->lll.period = (uint64_t)period * (1280000U / 625U);
+	scan->lll.duration = (uint64_t)duration * (10000U / 625U);
+
+	if (scan->lll.duration && scan->lll.period
```

Comment:
This validation are only required for controller-only build, use conditional compile using CONFIG_BT_CTLR_PARAM_CHECK

Reply:
Please valid the parameters, before use. Move validation before line 134.

Reply:
Fixed

Reply:
Just remembered why it was after. We need to scale period and duration to comare them. Moved the check back after the scaling.